docs/node-operators/how-to/network-upgrade-from-ops-perspective.md

```diff
@@ -0,0 +1,442 @@
+# Upgrade procedure from the OPS perspective
+
+## Assumptions
+
+**NOTE**: Below instruction is prepared for Debian-like Linux operating systems. We also assume you are already running a validator node. The following assumption is, We are using systemd commands(`systemctl` and `journalctl`) to control binaries in our setup. If you are using something different, that article's commands may vary.
+
+Before you start, you have to prepare and keep in mind a few things:
+
+- `<VEGA-USER>`; the Linux user that runs vega, e.g., `vega`
+- `<USER-HOME>`; path to the `<VEGA-USER>` home directory, e.g., `/home/vega`
+- `<VEGA-NETWORK-HOME>`; the home path to the vega network, e.g., `/home/vega/vega_home`
+- `<DATA-NODE-HOME>`; the home path to the data-node, the home can be safely shared with vega core, e.g., `/home/vega/data_node_home`
+- `<TENDERMINT-HOME>`; the tendermint home path, e.g., `/home/vega/tendermint_home`
+- `<VEGAVISOR-HOME>`; the vegavisor home path, e.g., `/home/vega/vegavisor_home`
+- `<BACKUP-FOLDER>`; the folder where you store backups, e.g., `/home/vega/backups`
+- `<VISOR-BIN>`; the path to the vega visor binary, e.g., `/home/vega/bin/visor`
+- `<VEGA-BIN>`; the path to the vega core binary, e.g., `/home/vega/bin/visor`
+- `<CHAIN-ID>`; new chain id for network, it is required to pass as an argument for data-node, e.g., current chain id on mainnet is: `vega-mainnet-0009`
+
+
+Placeholders for the PostgreSQL
+
+- `<VEGA-DB-USER>` - a postgreSQL user you create and put in the data-node config
+- `<VEGA-DB-NAME>` - a postgreSQL data-base name
+- `<VEGA-DB-PASS>` - a password for your `<VEGA-DB-USER>`
+
+We will refer to the above paths in the following article. The sample paths given above are just examples. We recommend setting your paths that align with the conventions adopted in your company.
+
+# Upgrade steps
+
+
+### 1. Download a new binaries
+
+Download new binaries from [the GitHub release](https://github.com/vegaprotocol/vega/releases) and unzip them, then you have to put them in the `<VEGA-BIN>` and the `<VISOR-BIN>`
+
+The binaries you need: 
+
+- vega binary

```

Comment:
Add info that you don't need a separate binary for data-node. We will use vega binary

Reply:
fixed